docs/UserGuide.md

```diff
@@ -19,21 +18,21 @@ MagusStock is a Java command-line interface (CLI) application designed for inven
 4. Run the file with the command `java -jar magusstock.jar`.
 
 ## Features
```

Comment:
Works for me in the terminal not sure whether it will crash anything on github ver, changed it because intellij was telling me that it 'cannot resolve anchor' when i was editing other parts of the UG.